src/components/Form/Dropdown.js

```diff
@@ -33,7 +33,7 @@ const DefaultSelectionSummary = ({selectedItems = [], multiSelect, noSelectionLa
         summaryString = selectedCount ? selectedItems[0][nameAttribute] : noSelectionLabel;
     }
 
-    return (<Fragment><span>{summaryString}</span><span className="RCB-select-arrow"></span></Fragment>);
+    return (<Fragment><span className="RCB-dd-label-ellipsis">{summaryString}</span><span className="RCB-select-arrow"></span></Fragment>);
```

Comment:
Just use classname `RCB-dd-label` as whether you have ellipsis or not is to be handled in CSS

Reply:
Done